docs/index.html

```diff
@@ -371,340 +337,685 @@ <h3>RP Meta-data</h3>
+                combined with rate-limiting and other mitigations, a brute-force attack becomes infeasible.
+            </p>
+
+            <p>
+                The <a href="#siop">SIOP</a> SHOULD then continuously poll the <code>verification_uri</code>
+                endpoint with the <code>device_code</code> and <code>user_code</code> as detailed in
+                <a href="#md-device-binding-request">Device Binding Request</a> and
+                <a href="#md-device-binding-response">Device Binding Response</a>, until the user completes the
+                interaction, the <code>user_code</code> or <code>device_code</code> expires, or another error occurs.
+            </p>
+
+            <p>
+                <a href="#rp">RPs</a> supporting this specification MUST implement a user interaction sequence that
+                allows users to supply the <code>user_code</code> at some stage during the interaction. Other than that,
+                the exact sequence and implementation of the user interaction is up to the <a href="#rp">RP</a>.
+            </p>
+
+            <h4 id="md-device-binding-request">Device Binding Request</h4>
+            <p>
+                The <a href="#siop">SIOP</a> sends the <code>&lt;Device Binding Request&gt;</code> to the
+                verification URI. Between polling requests to the verification URI, the <a href="#siop">SIOP</a>
+                considers the device binding parameters enclosed in the <code>&lt;SIOP DID Request&gt;</code>
+                which are detailed in <a href="#md-generate-siop-request">Request (Multi-device Flow)</a>.
+            </p>
+            <p>
+                All requests from the <a href="#siop">SIOP</a> MUST use the Transport Layer Security
+                (TLS) protocol [[RFC8446]] and implement the best practices of BCP 195 [[RFC7525]].
+            </p>
+            <p>
+                The verification URI endpoint MUST support HTTP POST and MUST accept the
+                <code>&lt;Device Binding Request&gt;</code> provided as a JSON object with <code>Content-Type</code>
+                of the HTTP request set to <code>application/json</code>.
+            </p>
+            <p>
+                The <code>&lt;Device Binding Request&gt;</code> MUST contain the following parameters:
+            </p>
+            <ul>
+                <li>
+                    REQUIRED. <code>device_code</code> MUST contain the device verification code as described
+                    in <a href="#md-generate-siop-request">Request (Multi-device Flow)</a>.
+                </li>
+
+                <li>
+                    REQUIRED. <code>user_code</code> MUST contain the user code as described in
+                    <a href="#md-device-binding">Device Binding</a>.
+                </li>
+                <li>
+                    REQUIRED. <code>expires_in</code> MUST contain the expiration date of the <code>user_code</code>
```

Comment:
Inconsistency betwen "date" and "in seconds" -- also unclear what happens to this across multiple (repeated) polling requests. Should the client iteratively decrement, if it's a count (in seconds)?